kimchi/src/circuits/lookup/constraints.rs

```diff
@@ -1,6 +1,9 @@
 use crate::{
     circuits::{
-        expr::{prologue::*, Column, ConstantExpr},
+        expr::{
+            prologue::{witness, E},
```

Comment:
btw prologue is supposed to be prelude here (I think I was mixing up the two words), and as such is supposed to be imported with a wildcard (that's what `prelude`s are for)

Reply:
Done https://github.com/o1-labs/proof-systems/pull/752/commits/e74530b7cf67a9cf66787be09ff530dff090630e